internal/markdown/markdown.go

```diff
@@ -137,7 +137,7 @@ func FromMessages(client *slackclient.SlackClient, history *slackclient.HistoryR
 		if includeSpeakerHeader {
 			fmt.Fprintf(b, "> **%s** at %s\n",
 				username,
-				messageTime.Format("2006-01-02 15:04"))
+				messageTime.UTC().Format("2006-01-02 15:04"))
```

Comment:
Forcing the timestamp to a particular time zone feels fine to me, given the time zone is obvious in the output. The formatted string that's produced, does it indicate the time zone? If not, what about tacking on " UTC" at the end?

If we could force times to the calling user's time zone, that'd be perhaps better, but I don't know that we can easily get that, so just going with UTC feels fine, too.

Reply:
Right, so actually before this change it looks like:

> **rneatherway** at 2023-04-27 19:06
>
> Test message

(my local timezone is BST i.e. GMT+1)

and after:

> **rneatherway** at 2023-04-27 18:06
>
> Test message

now it's UTC but no indication of that.

<hr>

I'm actually a bit unsure what is best here. Perhaps to just use the local user's timezone and include it in the output? But that actually makes the tests a bit annoying unless we stub the timezone. That might be the best option.

Reply:
Ooh yeah if we can avoid changing the time zone by just stubbing the test, I think that would be preferable/least surprising.

Reply:
OK, we now include the timezone:

> **rneatherway** at 2023-04-27 19:06 BST
>
> Test message

(and it's stubbed to UTC for the tests)

This preserves the behaviour before this PR except that it is now clearer to the reader because the timezone is actually included!